app/src/apiClient.js

```diff
@@ -1,6 +1,6 @@
 export const getContacts = () => _get("/api/contacts");
 
-export const addContact = (name) => _post("/api/contacts", { name });
```

Comment:
This is so strange. This is your template code. But I know I had re-written it and had it working before. I added many contacts that way before I moved on. Thank you!